File: UpperCase/UpperCase.scala
object UpperCase{

    def main(args: Array[String]){
        println("Input string (split by space):")
        val str = scala.io.StdIn.readLine()
        var strs = str.split(" ")
        strs = strs.map(_.toUpperCase)
        strs.map(println(_))
    }

}

File: UpperCase/UpperCase.java
import java.io.*;

public class UpperCase{

    public static void main(String[] args){
        System.out.println("Input text:");
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in ));
        String read = null;
        try {
            read = br.readLine();
        } catch (IOException e) {
            e.printStackTrace();
        }
        String[] strs = read.split(" ");
        int index = 0;
        for(String s : strs){
            strs[index++] = s.toUpperCase();
        }
        for(String s : strs){
            System.out.println(s);
        }

    }

}
